deploy/configs/packages.yaml

```diff
@@ -74,6 +74,8 @@ packages:
       prefix: /usr
   petsc:
     variants: ~fortran
+  py-neurodamus:
+    variants: +all_deps
   python:
```

Comment:
as Matthias pointed out, this should be moved to applications_hpc.yaml under packages key (see NSE example)